src/js/experimental/media-chrome-selectmenu.js

```diff
@@ -1,34 +1,47 @@
 import '../media-chrome-button.js';
 import './media-chrome-listbox.js';
 import { window, document } from '../utils/server-safe-globals.js';
+import { closestComposedNode } from '../utils/element-utils.js';
 import { MediaStateReceiverAttributes } from '../constants.js';
 
 const template = document.createElement('template');
 template.innerHTML = `
   <style>
   :host {
-    display: inline-block;
+    display: inline-flex;
+    position: relative;
+    flex-shrink: .5;
   }
 
-  media-chrome-button:not(:focus-visible) {
-    outline: none;
+  [name="listbox"]::slotted(*),
+  media-chrome-listbox {
+    position: absolute;
+    left: 0;
+    bottom: 100%;
+    max-height: 300px;
+    overflow: hidden auto;
   }
   </style>
 
-  <media-chrome-button aria-haspopup="listbox">
-    <slot name="button-content"></slot>
-  </media-chrome-button>
+  <slot name="button">
+    <media-chrome-button aria-haspopup="listbox">
+      <slot name="button-content"></slot>
+    </media-chrome-button>
+  </slot>
   <slot name="listbox" hidden>
     <media-chrome-listbox id="listbox" part="listbox">
       <slot></slot>
     </media-chrome-listbox>
   </slot>
 `;
 
-class MediaChromeMenuButton extends window.HTMLElement {
+class MediaChromeSelectmenu extends window.HTMLElement {
```

Comment:
thought(non-blocking): because nothing can be predictable 😞 https://chromium.googlesource.com/chromium/src/+/eaf93b7df0ff4c73760e56d40a54fc740841cd16/third_party/blink/renderer/core/html/forms/html_select_menu_element.h#23 casing is `HtmlSelectMenuElement`... just putting it on your radar. Defer to you on preferred casing for us.

Reply:
why do they do this???

Reply:
decided to follow suit.